libs/coin-polkadot/package.json

```diff
@@ -29,13 +29,46 @@
         "lib-es/*"
       ],
       "*": [
-        "lib/*"
+        "lib/*",
+        "lib/bridge/*",
+        "lib/logic/*",
+        "lib/signer/*",
+        "lib/test/*",
+        "lib/types/*"
       ]
     }
   },
   "exports": {
     "./lib/*": "./lib/*.js",
     "./lib-es/*": "./lib-es/*.js",
+    "./deviceTransactionConfig": {
```

Comment:
Is to ease the access of this file.
Instead of
```import deviceTransactionConfig from "@ledgerhq/coin-polkadot/bridge/deviceTransactionConfig"```

You can write
```import deviceTransactionConfig from "@ledgerhq/coin-polkadot/deviceTransactionConfig"```

My goal for one next PR is to avoid knowing the path of what you want to import, but limit public access